### ISSUE TITLE
Import error ModuleNotFoundError: No module named 'requests'

### ISSUE THREAD

**Issue by nadiiach** [bethgelab/foolbox#562]
I just installed foolbox using `pip install foolbox` but when I am trying to import it I get the following exception:

```
Python 3.6.10 |Anaconda, Inc.| (default, May  8 2020, 02:54:21) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.

import foolbox as fb

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/afs/csail.mit.edu/u/n/nadiia/.local/lib/python3.6/site-packages/foolbox/__init__.py", line 38, in <module>
    from . import zoo  # noqa: F401
  File "/afs/csail.mit.edu/u/n/nadiia/.local/lib/python3.6/site-packages/foolbox/zoo/__init__.py", line 2, in <module>
    from .weights_fetcher import fetch_weights  # noqa: F401
  File "/afs/csail.mit.edu/u/n/nadiia/.local/lib/python3.6/site-packages/foolbox/zoo/weights_fetcher.py", line 1, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

I double-checked that all requirements are satisfied.

**Comment by jonasrauber** [comment]
The error is right there: `ModuleNotFoundError: No module named 'requests'`
So `pip install requests` should fix it.

**Comment by jonasrauber** [comment]
Related to #534. We should certainly improve the error message.

**Comment by jonasrauber** [comment]
This was closed by #572
